src/BlazorWebFormsComponents/Interfaces/IColumn.cs

```diff
@@ -14,5 +16,6 @@ public interface IColumn<ItemType>
 	/// The parent IColumnCollection where the IColumn resides
 	/// </summary>
 	IColumnCollection<ItemType> ParentColumnsCollection { get; set; }
+		RenderFragment Render(ItemType item);
```

Comment:
Just remove extra spaces before

Reply:
I'll run a dotnet format and clean this up...  thank you for pointing that out